qcore/timeseries.py

```diff
@@ -368,6 +378,17 @@ def all2txt(self, prefix='./', dt=None):
         for s in self.stations.name:
             self.vel2txt(s, prefix=prefix, title=prefix, dt=dt)
 
+    def all2acc(self, prefix='./', dt=None):
```

Comment:
all2txt does not imply acc or vel.
therefore instead of copying function differing only by vel2txt, acc2txt, add option that sets func = vel2txt or acc2txt based on parameter